### ISSUE TITLE
No image preview for MeteoSwiss objects

### ISSUE THREAD

**Issue by daria-sl** [globe-swiss/phaenonet-client#169]
MeteoSwiss objects will never have images, so we should delete photo placeholders for them (pop-up on map). 
![grafik](https://user-images.githubusercontent.com/65400300/130772278-e463d9c9-8deb-45d3-bfda-e321fa09d80a.png)
Design see https://xd.adobe.com/view/3c0208f7-239e-489b-a049-7622c6fc4016-6951/screen/80e74fb9-f4e4-4e29-9d12-1c569460ccf8 slide 9

**Comment by pgoellnitz** [comment]
wouldn't it be great if MeteoSwiss would provide station pictures instead :)

**Comment by daria-sl** [comment]
@pgoellnitz yes you are right, that would be great. However, these pictures do not exist. I asked MeteoSwiss if they plan to provide them at some point but didn't get a response so far. I guess chances are really small... so I'd still propose to delete the placeholder for MeteoSwiss objects.

**Comment by pgoellnitz** [comment]
If they can't provide images for their station I would propose one generic 
image for all MeteoSwiss stations.

I would appreciate if we could avoid introducing new layout variants.

**Comment by daria-sl** [comment]
Good idea. I think we should let Dani design such a generic image together with the MeteoSwiss icon proposed in issue https://github.com/globe-swiss/phaenonet-client/issues/174 (introducing a photo would be misleading in my opinion, as the photo could never be representative of all stations). The question of how to prioritize this remains. What is your oppinion? I guess Dani would have time right now to do this, as we are awaiting more information from WSL.

**Comment by pgoellnitz** [comment]
I don't think that this is critical in any sense. I would add this topic to 
the layout improvement story and prioritize it there.
But it's a quick fix not longer than 1h I would guess.

**Comment by daria-sl** [comment]
> 
> I don't think that this is critical in any sense. I would add this topic to the layout improvement story and prioritize it there. But it's a quick fix not longer than 1h I would guess.

okay, done =)
